src/nginx/grpc.cc

```diff
@@ -76,6 +76,42 @@ std::pair<Status, std::string> GrpcGetBackendAddress(
       Status(NGX_DECLINED, "No GRPC backend address specified"), std::string());
 }
 
+std::shared_ptr<::grpc::ChannelCredentials> CreateChannelCredentials(
+    ngx_http_request_t *r, ngx_esp_loc_conf_t *espcf) {
+  if (espcf->grpc_backend_ssl == nullptr) {
+    ngx_log_error(NGX_LOG_DEBUG, r->connection->log, 0,
```

Comment:
Are these all errors? 

All of the logging statements in this function are logged with `ngx_log_error`

Reply:
it is only log as debug.  don't be confused with ngx_log_error.  it is NGX_LOG_DEBUG.

Reply:
I see -- it seems like `ngx_log_error` is used for non-errors as well. 

FYI: There is also `ngx_log_debug` which is just an alias to `ngx_log_error`
https://www.nginx.com/resources/wiki/extending/api/logging/#ngx-log-debug